packages/backend/src/open_payments/receiver/model.ts

```diff
@@ -47,6 +43,10 @@ export class Receiver {
       throw new Error('Cannot create receiver from expired incoming payment')
     }
 
+    if (!incomingPayment.methods.length) {
```

Comment:
moved this below the payment completed or expired check, to provide clearer error when creating outgoing payment